packages/@aws-cdk/aws-cloudtrail/README.md

```diff
@@ -72,3 +72,20 @@ trail.addS3EventSelector(["arn:aws:s3:::foo/"], {
   readWriteType: ReadWriteType.ALL,
 });
 ```
+
+For using CloudTrail event selector to log events about Lambda
+functions, you can use `addLambdaEventSelector`.
+
+```ts
+import cloudtrail = require('@aws-cdk/aws-cloudtrail');
+import lambda = require('@aws-cdk/aws-lambda');
+
+const trail = new cloudtrail.Trail(this, 'MyAmazingCloudTrail');
+const lambdaFunction = new lambda.Function(this, 'AnAmazingFunction);
```

Comment:
You also have a syntax error here. I'm going to guess you did not try to compile this :)

Reply:
Symptom of always having Prettier around in other projects 😄 

Will address today along with the test resolutions.